fluentd-gcp-image/Gemfile

```diff
@@ -1,8 +1,8 @@
 source 'https://rubygems.org'
 
-gem 'fluentd', '~>0.12.32'
+gem 'fluentd', '~>1.2.5'
```

Comment:
Note: we have only qualified this version of fluentd as it pertains to the Docker image we distribute (i.e., the one built from https://github.com/GoogleCloudPlatform/google-fluentd/blob/master/docker/Dockerfile). There is no evidence that it will work properly for this Docker image. Should this not be obsoleted now that we are providing one?
@qingling128 @bmoyles0117 FYI.